src/System.Management.Automation/engine/regex.cs

```diff
@@ -148,15 +148,8 @@ private void Init()
                 return;
```

Comment:
@iSazonov, your last commit had 498 failures in `PowerShell-CI-windows`
(These are 5 of the failures)

Enable-ExperimentalFeature and Disable-ExperimentalFeature tests.Error occurred in Describe block
```powershell
Cannot find drive. A drive with the name 'TestRegistry' does not exist.
At D:\a\1\s\src\powershell-win-core\bin\Release\netcoreapp3.0\win7-x64\publish\Modules\Pester\4.8.0\Functions\TestRegistry.ps1:60 char:47
```
Validate about_help.txt under culture specific folder works.Error occurred in Describe block
```powershell
Cannot find drive. A drive with the name 'TestRegistry' does not exist.
At D:\a\1\s\src\powershell-win-core\bin\Release\netcoreapp3.0\win7-x64\publish\Modules\Pester\4.8.0\Functions\TestRegistry.ps1:60 char:47
```
Get-Help -Online opens the default web browser and navigates to the cmdlet help content.Error occurred in Describe block
```powershell
Cannot find drive. A drive with the name 'TestRegistry' does not exist.
At D:\a\1\s\src\powershell-win-core\bin\Release\netcoreapp3.0\win7-x64\publish\Modules\Pester\4.8.0\Functions\TestRegistry.ps1:60 char:47
```
Testing Get-Help Progress.Error occurred in Describe block
```powershell
Cannot find drive. A drive with the name 'TestRegistry' does not exist.
At D:\a\1\s\src\powershell-win-core\bin\Release\netcoreapp3.0\win7-x64\publish\Modules\Pester\4.8.0\Functions\TestRegistry.ps1:60 char:47
```
Language Primitive Tests.Error occurred in Describe block
```powershell
Cannot find drive. A drive with the name 'TestRegistry' does not exist.
At D:\a\1\s\src\powershell-win-core\bin\Release\netcoreapp3.0\win7-x64\publish\Modules\Pester\4.8.0\Functions\TestRegistry.ps1:60 char:47
```